src/App.tsx

```diff
@@ -1,26 +1,85 @@
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-import { Loader, TodoFilter, TodoList, TodoModal } from './components';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { TodoModal } from './components/TodoModal';
+import { Loader } from './components/Loader';
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+import { FilterStatus } from './utils/FilterStatus';
+import { getTodos } from './api';
+
+import { Todo } from './types/Todo';
+import { useAppDispatch, useAppSelector } from './features/hooks';
+import { actions as todoActions } from './features/todos';
+
+function handleFiltration(data: Todo[], query: string, filterBy: string) {
+  let visibleData = [...data];
+
+  if (filterBy) {
+    switch (filterBy) {
+      case FilterStatus.ACTIVE:
+        visibleData = [...data].filter(todo => !todo.completed);
+        break;
+      case FilterStatus.COMPLETED:
+        visibleData = [...data].filter(todo => todo.completed);
+        break;
+    }
```

Comment:
Consider adding a default case in the switch statement to handle unexpected filter statuses. This will help prevent potential bugs if a new filter status is introduced or if the current filter status is not recognized.